website/project/views/register.py

```diff
@@ -43,6 +45,41 @@ def node_register_page(auth, **kwargs):
     out.update(_view_project(node, auth, primary=True))
     return out
 
+@must_be_valid_project
+@must_have_permission(ADMIN)
+@must_be_public_registration
+def node_registration_retraction(auth, **kwargs):
+    """ Handles retraction of public registrations
+
+    :param auth: Authentication object for User
+
+    :return: Template for GET, redirect URL for successful POST
+    """
+
+    node = kwargs['node'] or kwargs['project']
+
+    if request.method == 'POST':
+
+        data = request.json
```

Comment:
We might want to take some action here to remove retracted registrations from search results (or at least remove hidden content like wiki pages).